src/pages/genes/network/filters/index.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import Slider from '../../../../components/slider';
+import Tooltip from '../../../../components/tooltip';
+
+import { ReactComponent as Genes } from '../../../../images/genes.svg';
+import { ReactComponent as Link } from '../../../../images/link.svg';
+
+import './index.css';
+
+const Filters = ({
+  selectedNodes,
+  filteredNodes,
+  filteredLinks,
+  fullNodes,
+  fullLinks,
+  nodeCutoff,
+  setNodeCutoff,
+  edgeWeightCutoff,
+  setEdgeWeightCutoff
+}) => (
+  <>
+    <Slider
+      title='Node cutoff'
+      value={nodeCutoff}
+      min={selectedNodes}
+      max={fullNodes}
+      step={1}
+      onChange={setNodeCutoff}
+    />
+    <Slider
+      title='Edge weight cutoff'
+      value={edgeWeightCutoff}
+      min={0.4}
```

Comment:
The minimum should be the `g2g_edge_cutoff` value of selected mlmodel, see:
https://py3-adage.greenelab.com/api/v1/model/